.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: python
+env:
+  - PYTHON_VERSION=2.7
```

Comment:
I just need Py2 support temporarily for the taxonomy evaluation framework which I need to update to Py3. We can drop this once we need an skbio import (there's no skbio dependency at the moment).